### PR TITLE
Thats a lot of updates

### PR DESCRIPTION
**Pull request by ScribbleTAS**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,22 +1,45 @@
...
+To build everything select the `LoTASFabric->build->build`
+You will find the versions under the 1.14.4-1.18.1 folders under LoTAS-Fabric/version2/mcversion/build/libs
+
 ## Pushing
-Before Pushing, please switch to 1.12.2 again
\ No newline at end of file
+Before Pushing, please switch to 1.12.2/1.14.4 again
+
+## Troubleshooting
+Yes we do understand that this setup is pretty jank... If you need help you can contact us under this discord https://discord.gg/sdMc5UrGPN and look for Pancake or Scribble
```
Comment (PancakeTAS) [inline CONTRIBUTING.md]: That should not be there.

**Review by PancakeTAS — CHANGES_REQUESTED** [review]
See CONTRIBUTING.md L41 in my review

File: CONTRIBUTING.md (hunk 1 above)
Comment (ScribbleTAS) [reply CONTRIBUTING.md]: Where should it be then?

**Review by PancakeTAS — APPROVED** [review]
Works. Will merge later

File: CONTRIBUTING.md (hunk 1 above)
Comment (PancakeTAS) [reply CONTRIBUTING.md]: Nowhere

**Pull request merged**